DisCatSharp.Docs/articles/important_changes/10_0_0.md

```diff
@@ -0,0 +1,24 @@
+---
+uid: important_changes_10_0_0
+title: Version 10.0.0
+---
+
+# Upgrade from **9.9.0** to **10.0.0**
+
+## What is new in DisCatSharp?
+- Advanced dependency injection system
+- Support for API v10
+- Message content intent
+- Properly working application command localization
+- Optimized lib code
+- Pre-implementation of upcoming things
+- Support for [Channel Type](xref:DisCatSharp.Enums.ChannelType) `Forum`
+
+## What changed?
+
+To get message content with API v10, you have to enable the message content intent in the developer portal AND to specify the [DiscordIntent](xref:DisCatSharp.DiscordIntents) `MessageContent`. Otherwise you won't receive message contents from guild messages were the bot isn't mentioned.
```

Comment:
"AND `to` specify" => "AND specify"
"were the bot isn't mentioned" => "w`h`ere the bot isn't mentioned"

Also, isn't the message intents thing solely necessary for very large (verified?) bots? Should be noted, those probably make up a tiny minority of DCS' userbase.